simpleLooping/incrementBinaryList.py

```diff
@@ -18,3 +18,33 @@
 #   incrementBinaryList([1, 0, 0, 1]) # Return [0, 1, 0, 1]
 #   incrementBinaryList([1, 1, 0, 1]) # Return [0, 0, 1, 1]
 #   incrementBinaryList([1, 1, 1, 1]) # Return [0, 0, 0, 0, 1]
+from typing import List
+
+def incrementBinaryList(lst: List[int]) -> List[int]:
+    if len(lst) == 0:
+        return -1
+    
+    for i in lst:
+        if i != 0 and i != 1:
+            return -1
+
+    lenLst = len(lst)
```

Comment:
Why you don’t bring this one to the beginning so that “if len(lst) == 0” can use it right away. Remember len(lst) can be really slow for big list. So, any improvement is appreciate.